PowerValidatedSolutions.psm1

```diff
@@ -48145,6 +48145,383 @@ Function Test-WMSubnetInput {
 }
 Export-ModuleMember -Function Test-WMSubnetInput
 
+Function Invoke-GeneratePrivateKeyAndCsrFiles {
+    <#
+        .SYNOPSIS
+        Generate Private Key and CSR files using Openssl, common name and neccesary parameters.  
+
+        .DESCRIPTION
+        The Invoke-GeneratePrivateKeyAndCsrFiles cmdlet generate private key and CSR files using Openssl, common name and input parameters.  
+        
+        .EXAMPLE
+        Invoke-GeneratePrivateKeyAndCsrFiles -certDir "C:\certificates" -commonName "xint-vrli01.rainpole.io" -subjectAlternativeNames 
+        "xint-vrli01a.rainpole.io, xint-vrli01b.rainpole.io, xint-vrli01c.rainpole.io" -keySize 4096 -expireDays 720 -organization rainpole 
+        -organizationUnit "IT" -locality "San Francisco" -state "California" -country "US"
```

Comment:
```suggestion
        Invoke-GeneratePrivateKeyAndCsrFiles -certDir "C:\certificates" -commonName "sfo-vrli01.sfo.rainpole.io" -subjectAlternativeNames 
        "sfo-vrli01a.sfo.rainpole.io, sfo-vrli01b.sfo.rainpole.io, sfo-vrli01c.sfo.rainpole.io" -keySize 4096 -expireDays 720 -organization rainpole  -organizationUnit "IT" -locality "San Francisco" -state "California" -country "US"
```

Reply:
Addressed in commit 53039acbaaffafaf219274cac1a3684735130f53